yamlcrd/pkg/parser/testdata/simple1/expected.yaml

```diff
@@ -0,0 +1,70 @@
+XNamespace:
```

Comment:
it's hard to compare input with output since none of your fields appear to be omitempty in the output.

Reply:
Good point!  And fixed - I was importing the wrong yaml library (yaml.v3 instead of sigs.k8s.io/yaml)